server.js

```diff
@@ -97,6 +97,24 @@ module.exports = function(opts, callback) {
       app.use(prefix, express.static(path.join(__dirname, "public")));
       app.use(prefix, express.static(path.join(__dirname, "bower_components")));
       app.use(prefix, serve(tilelive, config[prefix]));
+      tilelive.load(config[prefix].source, function(err, src) {
+        if (err) {
+          throw err;
+        }
+
+        return tessera.getInfo(src, function(err, info) {
+          if (err) {
+            debug(err.stack);
+            return;
+          }
+
+          if (info.format === "pbf") {
+            app.use(prefix + "/_", serve(tilelive, "xray+" + config[prefix].source));
+            app.use(prefix + "/_", express.static(path.join(__dirname, "public")));
+            app.use(prefix + "/_", express.static(path.join(__dirname, "bower_components")));
+          }
+        });
+      });
```

Comment:
Where is the code that this replaces? This feels like it duplicates something now...